app/helpers.php

```diff
@@ -67,6 +67,22 @@ function subscribers_enabled()
     }
 }
 
+if (!function_exists('direct_notifications_enabled')) {
+    /**
+     * Is the direct notification functionality enabled and configured.
+     *
+     * @return bool
+     */
+    function direct_notifications_enabled()
```

Comment:
There's a `subscribers_enabled()` method, I though it'd be good to follow the convention.
I am fine either way